aws/components/db/setup.ftl

```diff
@@ -773,6 +792,15 @@
                         enhancedMonitoringRoleId=monitoringRoleId!""
                         performanceInsights=solution.Monitoring.QueryPerformance.Enabled
                         performanceInsightsRetention=solution.Monitoring.QueryPerformance.RetentionPeriod
+                        maintenanceWindow=
+                            (solution.MaintenanceWindow.DayOfTheWeek)?has_content?then(
```

Comment:
You can use the isConfigured function on solution.MaintenanceWindow

That instead checks to see if the MaintenanceWindow object is present on the component configuration and has been added somewhere along the line

Reply:
I must be doing something wrong. I don't find a isConfigured function. Can you point me at it?